setup.py

```diff
@@ -15,7 +15,7 @@
     description="Python tools for Synthetic Biology.",
     long_description=long_description,
     long_description_content_type="text/markdown",
-    url="https://github.com/Global-Biofoundries-Alliance/SynBioPython",
+    url="https://biopython.org",
```

Comment:
http://synbiopython.org/

Reply:
Thanks, @vegh. I've updated the redirect such that both http and https redirect successfully.

Reply:
Hi Neil, as pointed by Peter, I think the url in the setup.py file needs to be changed to "http://synbiopython.org" instead of biopython.org. Everything else looked good to me. We can merge the copy after this.

Reply:
Sorry. That's a little embarrassing as well. Thanks!